janusgraph-cql/pom.xml

```diff
@@ -0,0 +1,342 @@
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <phase>test</phase>
+                        <configuration>
+                            <excludedGroups>${test.excluded.groups},org.janusgraph.testcategory.OrderedKeyStoreTests,org.janusgraph.testcategory.SerialTests,org.janusgraph.testcategory.CassandraSSLTests</excludedGroups>
+                            <groups />
+                            <reportNameSuffix>${test.murmur}</reportNameSuffix>
+                            <systemProperties>
+                                <property>
+                                    <name>test.cassandra.confdir</name>
+                                    <value>${project.build.directory}/cassandra/${test.murmur}/conf</value>
+                                </property>
+                                <property>
+                                    <name>test.cassandra.datadir</name>
+                                    <value>${project.build.directory}/cassandra/${test.murmur}/data</value>
+                                </property>
+                            </systemProperties>
+                            <skip>${test.skip.murmur}</skip>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>${test.murmur}-serial-test</id>
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <phase>test</phase>
+                        <configuration>
+                            <excludedGroups />
+                            <groups>org.janusgraph.testcategory.SerialTests</groups>
+                            <parallel>none</parallel>
+                            <perCoreThreadCount>false</perCoreThreadCount>
+                            <threadCount>1</threadCount>
+                            <runOrder>alphabetical</runOrder>
+                            <reportNameSuffix>${test.murmur}-serial</reportNameSuffix>
+                            <systemProperties>
+                                <property>
+                                    <name>test.cassandra.confdir</name>
+                                    <value>${project.build.directory}/cassandra/${test.murmur}/conf</value>
+                                </property>
+                                <property>
+                                    <name>test.cassandra.datadir</name>
+                                    <value>${project.build.directory}/cassandra/${test.murmur}/data</value>
+                                </property>
+                            </systemProperties>
+                            <skip>${test.skip.murmur-serial}</skip>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>${test.murmur-ssl}-test</id>
```

Comment:
Ah sorry about that. I was comparing to a cassandra log I had on hand but now I see in that case there was a failure in the bop-test execution so the others didn't run.